src/main/pages/setup-kubernetes/kubernetes-admin-guide.md

```diff
@@ -0,0 +1,208 @@
+tags: [installation, kubernetes]
+sidebar: user_sidebar
+permalink: kubernetes-admin-guide.html
+folder: setup-kubernetes
+---
+
+{% include links.html %}
+
+## Examples
+
+All examples use `kubectl` command. OpenShift admins should use `oc`
+
+## RAM
+
+**Single User**
+
+Che server pod consumes up to 1GB RAM. The initial request is 256MB, and server pod rarely consumes more than 800MB. A typical workspace will require 2GB. So, **<span style="color:red;">3GB</span>** is a bare minimum to try single user Che on OpenShift/Kubernetes.
+
+**Multi-User**
+
+Depending on whether or not you deploy Che bundled with Keycloak auth server and Postgres database, RAM requirements for a multi-user Che installation can vary.
+
+When deployed with Keycloak and Postgres, your Kubernetes cluster should have **at least <span style="color:red;">5GB RAM</span>** available - 3GB go to Che deployments:
+
+* ~750MB for Che server
+* ~1GB for Keycloak
+* ~515MB for Postgres)
+* min 2GB should be reserved to run at least one workspace. The total RAM required for running workspaces will grow depending on the size of your workspace runtime(s) and the number of concurrent workspace pods you run.
+
+## Resource Allocation and Quotas
+
+All workspace pods are created either in the same namespace with Che itself, a dedicated namespace, or a new namespace is created for every workspace. In both cases, pods are are created in the account of a user who deployed Che (if che service account is used to create ws objects) or account of a user whose token/credentials are used in Che deployment env.
+
+Therefore, an account where Che pods are created (including workspace pods) should have reasonable quotas for RAM, CPU and storage, otherwise, workspace pods won't be created.
+
+## Who Creates Workspace Objects?
+
+All workspace objects for all Eclipse Che users are **created on behalf of one OpenShift user**, typically the one who deployed Che or whose credentials or token is used in Che deployment environment. Eclipse Che uses fabric8 client to talk to Kubernetes/OpenShift API, and the client uses token, username/password or service account, if available.
+
+* If you passed `CHE_INFRA_KUBERNETES_USERNAME` and `CHE_INFRA_KUBERNETES_PASSWORD` to Che deployment, those credentials will be used to create workspace objects.
+
+* If there's `CHE_INFRA_KUBERNETES_OAUTH__TOKEN` fabric8 client will use it. If both are available, token has a priority over username/pass.
+
+* If none are available **che service account** is used to create workspace objects. This is default behavior. Since che SA cannot create objects outside a namespace it's bound to, all workspace objects are created in Che namespace. Admins can grant `che` [service account super user permissions](https://kubernetes.io/docs/admin/authorization/rbac/#service-account-permissions), and this way it will be possible to use this SA to create objects outside Che namespace. OpenShift cluster admins can do it this way using oc CLI:
```

Comment:
I don't understand what "Since che SA cannot..." means - is that a typo?

Reply:
SA == service account. Fixed.